src/VirtualClient/VirtualClient.Contracts/Metric.cs

```diff
@@ -126,6 +136,11 @@ public Metric(string name, double value, string unit, MetricRelativity relativit
         /// </summary>
         public IDictionary<string, IConvertible> Metadata { get; }
 
+        /// <summary>
+        /// Metric verbosity to descript importance of metric.
+        /// </summary>
+        public MetricVerbosity Verbosity { get; set; } = MetricVerbosity.Standard;
```

Comment:
Same as noted earlier. Let's use the out of box enum we use otherwise for describing logging level/severity: LogLevel. We don't need an extra enum abstraction.